### PR TITLE
Add ids to homeserver and passphrase fields

### PR DESCRIPTION
**Pull request by SimonBrandner**
This is useful because some password managers allow you to specify custom fields to auto-fill

### CODE REVIEW

File: src/components/views/dialogs/ServerPickerDialog.tsx [hunk 1]
```diff
@@ -217,6 +217,7 @@ export default class ServerPickerDialog extends React.PureComponent<IProps, ISta
                         value={this.state.otherHomeserver}
                         validateOnChange={false}
                         validateOnFocus={false}
+                        id="homeserverInput"
```
Comment (t3chguy) [inline src/components/views/dialogs/ServerPickerDialog.tsx:220]: I think we typically prefix identifiers with `mx_`

File: src/components/views/dialogs/security/AccessSecretStorageDialog.tsx [hunk 2]
```diff
@@ -345,6 +345,7 @@ export default class AccessSecretStorageDialog extends React.PureComponent<IProp
                 <form className="mx_AccessSecretStorageDialog_primaryContainer" onSubmit={this.onPassPhraseNext}>
                     <input
                         type="password"
+                        id="passPhraseInput"
```
Comment (SimonBrandner) [inline src/components/views/dialogs/security/AccessSecretStorageDialog.tsx:348]:
```suggestion
                        id="mx_passPhraseInput"
```

File: src/components/views/dialogs/ServerPickerDialog.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/components/views/dialogs/ServerPickerDialog.tsx:220]: Good point
```suggestion
                        id="mx_homeserverInput"
```

**Review by t3chguy — APPROVED** [review]
looks sane to me

**Pull request merged**